force-app/main/default/triggers/TaskTrigger.trigger

```diff
@@ -0,0 +1,14 @@
+trigger TaskTrigger on Task(after insert, after update, after delete) {
+  if (Trigger.isInsert) {
+    TaskTriggerHandler.handleAfterInsert(Trigger.new);
+  } else if (Trigger.isUpdate) {
+    TaskTriggerHandler.handleAfterUpdate(Trigger.old, Trigger.new);
+  } else if (Trigger.isDelete) {
+    TaskTriggerHandler.handleAfterDelete(Trigger.old);
+  }
+
+  if (!Trigger.isAfter) {
+    return;
+  }
```

Comment:
These lines are useless. Please explain why

Reply:
Is that because in my trigger definition I only have the after events? And no before event, so the trigger won't fire before the record is inserted, updated etc.

`trigger TaskTrigger on Task(after insert, after update, after delete) {}`
I've tested the code without this lines and without the before cases among the trigger's params and everything proceeds normally: only the after events are handled.

`trigger TaskTrigger on Task(after insert, after update, after delete, before delete, before, update) {}`
The problem obviously arises when I add the before cases among the trigger's params (without doing any control on Trigger.isBefore() or Trigger.isAfter()).

So if in my trigger definition I only have the after events, I don't need this lines.

`trigger TaskTrigger on Task(after insert, after update, after delete) {}`

But shouldn't I do a trigger check anyway? In case before triggers will be added in the future? And since I have to create only one trigger per object in which I manage both after and before events.